packages/plugin_platform_interface/lib/plugin_platform_interface.dart

```diff
@@ -89,10 +100,10 @@ abstract class PlatformInterface {
       return;
     }
     if (preventConstObject &&
-        identical(instance._instanceToken, const Object())) {
```

Comment:
how did this code work before? I don't see a `noSuchMethod` anywhere. Wouldn't this have just thrown at runtime?

Reply:
It was previously wrong (see https://github.com/flutter/flutter/issues/109339 for some more discussion), and the language change was going to make it somewhat differently wrong in at least some cases. This makes it actually do what it was supposed to do, error-wise.

Reply:
All the more reason to test it. :-)

Reply:
Ok, test added!